opendebates/views.py

```diff
@@ -239,10 +239,15 @@ def vote(request, id):
 
         )
     )
+    previous_debate_time = get_previous_debate_time()
     if created:
         # update the DB with the real tally
         Submission.objects.filter(id=id).update(
             votes=F('votes')+1,
+            current_votes=F('current_votes')+(
+                1 if previous_debate_time is None or vote.created_at > previous_debate_time
```

Comment:
Not sure there's a need to increment this at all unless `previous_debate_time` is set and in the past; i.e., should the first part of that be `if previous_debate_time is not None and ...`?